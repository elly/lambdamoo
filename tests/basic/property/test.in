eval("add_property(#0, \"temp\", 0, { #3, \"rwc\" }); return 0;")
eval("add_property(#0, \"temp0\", 0, { #3, \"rwc\" }); return 0;")
eval("$temp = create(#1); return parent($temp);")
eval("$temp0 = create($temp); return parent($temp0) == $temp;")
eval("add_property($temp, \"test\", 1, { #3, \"rwc\" }); return 0;")
$temp.test
is_clear_property($temp0, "test")
eval("$temp0.test = 2; return $temp0.test;")
is_clear_property($temp0, "test")
$temp0.test
properties($temp)
properties($temp0)
eval("clear_property($temp0, \"test\"); return 0;")
is_clear_property($temp0, "test")
$temp0.test
