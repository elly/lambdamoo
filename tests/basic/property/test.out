{1, 0}
{1, 0}
{1, #1}
{1, 1}
{1, 0}
1
1
{1, 2}
0
2
{"test"}
{}
{1, 0}
1
1
